src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+private String value;
+	
+	public Block(String value){
+		this.value=value;
```

Comment:
@potatowagon Tabs are used for indentation in this file. Spaces should be used instead.